### ISSUE TITLE
xrootd.chksum script is not working for WebDAVs protocol on xrootd 5.5.2

### ISSUE THREAD

**Issue by geonmo** [xrootd/xrootd#1905]
Hello, XRootD Developers,

Our site (T3_KR_KISTI) uses xrootd to support the WebDAVs protocol. 

Since our backend storage is NFS storage, we configured the adler 32 checksum to be returned through a user script using the xrootd.chksum keyword to support checksum.
```
xrootd.chksum max 60000 adler32 /opt/checksum/adler.py
```

Since the recent xrootd 5.5.2 version upgrade, we have confirmed that the feature is not working well for webdavs. 

The test command is: 
```bash
gfal-sum -vvv -D "HTTP PLUGIN:RETRIEVE_BEARER_TOKEN=false" -t 90 davs://cms-se.sdfarm.kr:1094/xrd/store/mc/SAM/GenericTTbar/AODSIM/CMSSW_9_2_6_91X_mcRun1_realistic_v2-v1/00000/CE860B10-5D76-E711-BCA8-FA163EAA761A.root ADLER32
```

The error message is
```bash
gfal-sum error: 38 (Function not implemented) - [gfal2_checksum][gfal_http_checksum] checksum calculation for ADLER32 not supported for davs://cms-se.sdfarm.kr:1094/xrd/store/mc/SAM/GenericTTbar/AODSIM/CMSSW_9_2_6_91X_mcRun1_realistic_v2-v1/00000/CE860B10-5D76-E711-BCA8-FA163EAA761A.root
```

When we tested with the root protocol, we were able to check the value normally. 

When a checksum is requested by webdavs, it has been confirmed that the script works.

I think the related settings may have been changed, so I would like you to let me know about the related settings.

**Comment by abh3** [comment]
Hello Geonmo,

Indeed, you are not the first person to report this. There was a bug in 
the way we treated scripts that computed the checksum (as opposed to doing 
so natively) that returned a response gfal did not like. This will be 
corrected in 5.5.3 which is coming soon. So, you should stay with 5.5.1 or 
earlier for now.

Andrew

**Comment by geonmo** [comment]
@abh3 Thanks for your answer. I will close this issue.
